src/js_native_api_v8.cc

```diff
@@ -2483,13 +2481,10 @@ napi_status napi_instanceof(napi_env env,
 
   CHECK_TO_OBJECT(env, context, ctor, constructor);
 
-  if (!ctor->IsFunction()) {
-    napi_throw_type_error(env,
-                          "ERR_NAPI_CONS_FUNCTION",
-                          "Constructor must be a function");
-
-    return napi_set_last_error(env, napi_function_expected);
```

Comment:
Maybe JavaScript error should not be thrown here and returning `napi_function_expected` instead?

Reply:
That would be a semver-major change, I think, so we mustn't.

Reply:
Also, we cannot change the return status from `napi_function_expected` to `napi_pending_exception` because that's semver-major as well.